jwtauth.go

```diff
@@ -78,30 +100,45 @@ func (ja *JwtAuth) Handle(paramAliases ...string) func(chi.Handler) chi.Handler
 
 			// Token is required, cya
 			if tokenStr == "" {
-				err = errUnauthorized
+				err = ErrUnauthorized
 			}
 
 			// Verify the token
 			token, err := ja.Decode(tokenStr)
 			if err != nil || !token.Valid || token.Method != ja.signer {
-				http.Error(w, errUnauthorized.Error(), 401)
+				switch err.Error() {
+				case "token is expired":
```

Comment:
We shouldn't rely on `token.Valid` - it can result in false negatives: https://github.com/dgrijalva/jwt-go/blob/b728399c73fa4c9135df5ebddae4177583ab96f0/parser.go#L87-L90

`ja.IsExpired(token)` is safer